juniper_official/routing/check-isis-adjacency-status.rule

```diff
@@ -150,6 +150,12 @@
                                     releases 23.2R1 {
                                         release-support min-supported-release;
                                     }
+                                }
+                                platforms vmx {
```

Comment:
This will not work. As the product is not VMX.
Either put default sensors for operating-system junos or add a product for VMX